stubs/docutils/docutils/parsers/rst/languages/__init__.pyi

```diff
@@ -0,0 +1,15 @@
+from typing import ClassVar, Final, Protocol
+
+from docutils.languages import LanguageImporter
+
+__docformat__: Final = "reStructuredText"
+
+class _RstLanguageModule(Protocol):
```

Comment:
```suggestion
@type_check_only
class _RstLanguageModule(Protocol):
```

Reply:
thanks, i also added it to _LanguageModule(Protocol)